engine/phpcodefixer/meta.json

```diff
@@ -0,0 +1,11 @@
+{
+	"$schema": "https://repometric.github.io/linterhub/schema/meta.json",
+	"name": "phpcodefixer",
+	"description": "A console scanner that checks compatibility of your code with new interpreter versions.",
+	"url": "https://github.com/wapmorgan/PhpCodeFixer",
+	"languages": [
+		"PHP",
+		"HTML+PHP"
```

Comment:
`HTML+PHP` is not a language, fix this

Reply:
It's format `phtml`. We use `linguist` for identify languages and they extensions and for format `phtml` language is [`HTML+PHP`](https://github.com/repometric/linterhub/blob/master/schema/language.linguist.json#L1571).